api-docs/api-operations/methods/post-restart-ha-instance-version-accountid-ha-haid-action.rst

```diff
@@ -0,0 +1,148 @@
+
+.. _post-restart-ha-instance-version-accountid-ha-haid-action.rst:
+
+Restart a HA Instance
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code::
+
+    POST /{version}/{accountId}/ha/{haId}/action
+
+Restarts an HA instance. During the restart process, the HA Instance goes into a ``REBOOT state``. After the operation completes, the status changes to ``ACTIVE``.
+
+.. warning::
+   When the HA Instance Group is started, auto-failover is turned off (MHA stopped), then the mysql service on the replica nodes is restarted followed by the source node. Finally, auto-failover is turned on again. Refer to the Knowledge Center article for more details about MHA: `High Availability for Cloud Databases <https://www.rackspace.com/knowledge_center/article/high-availability-for-cloud-databases>`__.
+
```

Comment:
Replace "Refer to the ...." with the following:

See the :how-to:`High Availability for Cloud Databases <high-availability-for-cloud-databases>` article in the Rackspace How To collection.